commands-yml/commands/session/other/settings.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Update Device Settings
+short_description: Update the current setting on the device
+example_usage:
+  javascript_wd:
+    |
+      await driver.settings(settings);
+  javascript_wdio:
+    |
+      driver.updateSettings(settings);
+
+client_docs:
+  javascript_wdio: "http://webdriver.io/api/mobile/settings.html"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L3032"
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
+  mac:
+    mac: true
+  windows:
+    windows: true
+client_support:
+  java: true
+  python: true
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+  
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /wd/hub/session/:session_id/appium/settings
+  method: 'POST'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+  json_parameters:
+    - name: settings
+      type: string
+      description: Settings string
```

Comment:
Do we have any indication of what this "Settings string" might be?

Reply:
I'm not sure what that is tbh.

Reply:
Do you know anything about these commands @imurchie? It looks like they're not commonly used.

Reply:
See https://github.com/appium/appium-base-driver/blob/master/lib/basedriver/device-settings.js